helm/coredns-app/Chart.yaml

```diff
@@ -1,7 +1,9 @@
 annotations:
-  application.giantswarm.io/team: cabbage
+  io.giantswarm.application.team: cabbage
+  io.giantswarm.application.audience: "all"
+  io.giantswarm.application.managed: "true"
```

Comment:
we had a concurrency problem 🤣 https://github.com/giantswarm/coredns-app/pull/403#discussion_r2609976249

I'm not sure if we want to keep the managed annotation because of https://gigantic.slack.com/archives/C02GDJJ68Q1/p1765447609430609
we can merge now and remove it later though